build.gradle

```diff
@@ -2,45 +2,50 @@ buildscript {
     repositories {
         jcenter()
     }
+
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.1.2'
+        classpath 'com.android.tools.build:gradle:2.1.2'
         classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.0'
-        classpath 'com.github.dcendents:android-maven-plugin:1.2'
+        classpath 'com.github.dcendents:android-maven-gradle-plugin:1.3'
     }
 }
 
 apply plugin: 'com.android.library'
 apply plugin: 'com.github.dcendents.android-maven'
 apply plugin: 'com.jfrog.bintray'
 
-version = "0.9.6"
+version = "0.9.7"
 
 android {
-    compileSdkVersion 21
+    compileSdkVersion 24
```

Comment:
actually don't know if this is a good idea since 24 isn't technically released yet.  i think the api is finalized though, if this even matters at all